packages/giselle/src/engine/triggers/reconfigure-github-trigger.ts

```diff
@@ -32,6 +34,14 @@ export async function reconfigureGitHubTrigger(args: {
 		throw new Error("Only GitHub triggers are supported for updates");
 	}
 
+	const currentEvent = currentTrigger.configuration.event;
+	const requestedEvent = args.event ?? currentEvent;
+	if (requestedEvent.id !== currentEvent.id) {
+		throw new Error(
+			`Changing GitHub trigger event type is not supported (${currentEvent.id} → ${requestedEvent.id})`,
+		);
+	}
+
```

Comment:
Does this mean that the event cannot be changed with this implementation? The event is added as an optional argument, but won't an error always occur?